cypress/integration/nonSeededFlows/test.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('Describe for the new E2E build node', () => {
```

Comment:
A filler test for now to have the new E2E build node pass successfully. This will be replaced by admin onboarding tests @Ridhwana.